bin/pc2installservice

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+#
+# Purpose:
+#   This script installs a Linux service that runs the script to create
+#   the cgroup necessary for PC2 sandboxes.  The service is a one-shot service
+#   that simply runs the /usr/pc2/bin/pc2installsandbox shell script
+#
+# Author: John Buck
+
+#DEBUG=
+DEBUG="echo Would"
+
+SCRIPTNAME="$0"
+
+if test $# -ne 2
+then
+	echo Usage: $SCRIPTNAME pc2user pc2group
+	exit 4
```

Comment:
Personally, I strongly prefer defining constants for error exit codes (like EXIT_INCORRECT_NUM_ARGS).

Reply:
Noted.